internal/pkg/config/config.go

```diff
@@ -80,23 +83,28 @@ func InitConfig() {
 	viper.SetConfigType(configFileExtension)
 	viper.AddConfigPath(configFolderPath)
```

Comment:
One more thing, I think these are not needed anymore if we are using `SetConfigFile`, right? As this will override those values